src/PointwiseFunctions/AnalyticSolutions/Elasticity/HalfSpaceMirror.hpp

```diff
@@ -0,0 +1,190 @@
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <limits>
+
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Elliptic/Systems/Elasticity/FirstOrderSystem.hpp"
+#include "Elliptic/Systems/Elasticity/Tags.hpp"
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/Elasticity/ConstitutiveRelations/IsotropicHomogeneous.hpp"
+#include "PointwiseFunctions/Elasticity/ConstitutiveRelations/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+/// \cond
+class DataVector;
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace Elasticity::Solutions {
+/*!
+ * \brief The solution for a half-space mirror deformed by a laser beam.
+ *
+ * \details This solution is mapping (via the fluctuation dissipation theorem)
+ * thermal noise to an elasticity problem where a normally incident and
+ * axisymmetric laser beam with a Gaussian beam profile acts on the face of a
+ * semi-infinite mirror. Here we assume the face to be at \f$z = 0\f$ and the
+ * material to extend to \f$+\infty\f$ in the z-direction as well as for the
+ * mirror diameter to be comparetively large to the `beam width`. The mirror
+ * material is characterized by an isotropic homogeneous constitutive relation
+ * \f$Y^{ijkl}\f$ (see
+ * `Elasticity::ConstitutiveRelations::IsotropicHomogeneous`). In this scenario,
+ * the auxiliary elastic problem has an applied pressure distribution equal to
+ * the laser beam intensity profile (see Eq. (11.94) and Eq. (11.95) in
+ * \cite ThorneBlandford2017 with F = 1 and the time dependency dropped)
+ *
+ * \f{align}
+ * T^{zr} &= T^{rz} = 0 \\
+ * T^{zz} &= \frac{e^{-\frac{r^2}{r_0^2}}}{\pi r_0^2} \text{.}
+ * \f}
+ *
+ * in the form of a Neumann boundary condition to the face of the mirror. We
+ * find that this stress in cylinder coordinates is produced by the displacement
+ * field
+ *
+ * \f{align}
+ * \xi_{r} &= \frac{1}{2 \mu} \int_0^{\infty} dk J_1(kr)e^{(-kz)}\left(1 -
+ * \frac{\lambda + 2\mu}{\lambda + \mu} + kz \right) \tilde{p}(k) \\
+ * \xi_{\phi} &= 0 \\
+ * \xi_{z} &=  \frac{1}{2 \mu} \int_0^{\infty} dk J_0(kr)e^{(-kz)}\left(1 +
+ * \frac{\mu}{\lambda + \mu} + kz \right) \tilde{p}(k)
+ * \f}
+ *
+ * and the strain
+ *
+ * \f{align}
+ * \Theta &= \frac{1}{2 \mu} \int_0^{\infty} dk
+ * J_0(kr) k e^{(-kz)}\left(\frac{-2\mu}{\lambda + \mu}\right) \tilde{p}(k) \\
+ * S_{rr} &= \Theta - S_{\phi\phi} - S_{zz} \\
+ * S_{\phi\phi} &= \frac{\xi_{r}}{r} \\
+ * S_{(rz)} &= -\frac{1}{2 \mu} \int_0^{\infty} dk J_1(kr) k e^{(-kz)}\left(kz
+ * \right) \tilde{p}(k) \\
+ * S_{zz} &= \frac{1}{2 \mu} \int_0^{\infty} dk
+ * J_0(kr) k e^{(-kz)}\left(-\frac{\mu}{\lambda + \mu} - kz \right) \tilde{p}(k)
+ * \f}
+ *
+ * (see Eqs. (11 a) - (11 c) and (13 a) - (13 e), with (13 c) swapped in favor
+ * of (12 c) in \cite Lovelace2007tn), where \f$\tilde{p}(k)= \frac{1}{2\pi}
+ * e^{(\frac{kr}{2})^2}\f$ is the Hankel-Transform of the lasers intensity
```

Comment:
Did you plug in a Gaussian here? The Hankel transform is Eq. (5) of \Lovelace2007tn, which is an integral of \tilde{p}(k). Either way, please include the definition of \tilde{p}(k), along with what you are assuming for \tilde{p}(k)

Reply:
Can confirm.